submissions/Writerman/my_pop_up/css/style.css

```diff
@@ -0,0 +1,239 @@
+  width: 50px;
+  height: 50px;
+}
+.icon_defend
+{
+  background-image: url(../images/Defend.png);
+  width: 50px;
+  height: 50px;
+}
+.icon_atack {
+  background-image: url(../images/Atack.png);
+  width: 50px;
+  height: 50px;
+}
+.pop_up {
+  display: none;
+    position: absolute;
+    top: 7em;
+    right: 0.6em;
+    width: 350px;
+    border: 2px solid #666;
+    background: #fff;
+}
+.pop_up:before {
+  content: '';
+   display: block;
+   position: absolute;
+   top: -1.1em;
+   right: 32%;
+   width: 2em;
+   height: 2em;
+   border: 2px solid #666;
+   border-right:none;
+   border-bottom: none;
+   background: #fff;
+   transform: rotate(45deg);
+}
+.pop_up_list {
+  display: grid;
+  width: 100%;
+  height: 100%;
+  row-gap: 35px;
+  grid-template-columns: repeat(3, max-content);
+  justify-content: space-between;
+  padding: 40px;
+  overflow-y: auto;
+  box-sizing: border-box;
+}
+.pop_up_list::-webkit-scrollbar {
+  width: 2px;
+  background-color: rgba(102,102,102,0.1);
+}
+.pop_up_list::-webkit-scrollbar-thumb {
+  background-color: rgba(102,102,102,0.5);
+}
+.pop_up_item {
+  display: inline-block;
+}
+.app_link {
+  display: grid;
+  grid-template-rows: repeat(2, max-content);
+  text-decoration: none;
+  color: #666;
+  padding: 3px;
+}
+.app_link:hover,
+.app_link:focus {
+  opacity:0.5;
+  box-shadow: 10px -10px 22px 10px #000
+}
+.app_link:hover .app_name,
+.app_link:focus .app_name {
+  color: #03c;
+  text-decoration: underline;
+}
+.app_link_all_width {
+  grid-column: 1 / -1;
+}
+.app_icon {
+  display: block;
+  width: 50px;
+  height: 50px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position-x: center;
+  background-position-y: bottom;
+  justify-self: center;
+}
+.app_name {
+  display: block;
+  text-align: center;
+}
+.toggle_input {
+  position: absolute;
+  opacity: 0;
+  z-index: -1;
+}
+.toggle_input:checked ~ .pop_up {
+  display: block;
+}
+.icon_book {
+  background-image: url(../images/book.png);
+  width: 50px;
+  height: 50px;
+}
+.toggle_input:focus + .toggle_label > .icon_book {
+opacity:0.5;
+box-shadow: 10px -10px 22px 10px #000
+}
+
+.toggle_input:hover + .toggle_label >  .icon_book {
+  box-shadow: 10px -10px 22px 10px #000
+}
+.toggle_label {
+  cursor: pointer;
+}
+
+/* --extra-- */
+
+.button_extra_input {
+  position: absolute;
+  opacity: 0;
+  z-index: -1;
+}
+.button_extra_input ~ .app_link {
+  display: none;
+}
+.button_extra_input:checked ~ .app_link {
+  display: grid;
+}
+.button_extra_input:checked + .button_extra_label {
+  display: none;
+}
+.button_extra_input:checked {
+  display: none;
+}
+.button_extra_label {
+  display: block;
+  position: absolute;
+  bottom: 0;
+  left:0;
+  width: 100%;
+  text-align: center;
+  cursor: pointer;
+  background-color: greY;
+}
+.button_extra_input:hover + .button_extra_label,
+.button_extra_input:focus + .button_extra_label{
+  color: #03c;
+  text-decoration: underline;
+  opacity: 0.5;
+}
+.icon_first {
```

Comment:
This is also bad practice. Here and below. Better to name the icons essentially. Describing what the icon actually is.